tests/api/models/test_apps3bucket.py

```diff
@@ -56,7 +42,12 @@ def test_update_aws_permissions(app, bucket, aws):
 
     apps3bucket.save()
 
-    aws.put_role_policy.assert_called()
+    aws.grant_bucket_access.assert_called_with(
+        app.iam_role_name,
+        bucket.arn,
+        AppS3Bucket.READONLY,
+        apps3bucket.resources,
```

Comment:
I wonder if at some point we should rename this to something less ambiguous/confusing than `resources`.
It took us a bit to understand that this was referring to S3 bucket paths.

I like much more the `path_arns` name used in the `aws.grant_bucket_access()` method definition.